# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qt5/qfloat16.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- qt-everywhere-src-5.14.2/qtbase/src/corelib/global/qfloat16.cpp.vanilla	2020-05-17 21:58:04.029166720 +0200
+++ qt-everywhere-src-5.14.2/qtbase/src/corelib/global/qfloat16.cpp	2020-05-17 22:03:11.526165126 +0200
@@ -282,5 +282,5 @@
 
 #include "qfloat16tables.cpp"
 #ifdef QFLOAT16_INCLUDE_FAST
-#  include "qfloat16_f16c.c"
+//#  include "qfloat16_f16c.c"
 #endif
